packages/gestalt/src/Datapoint.js

```diff
@@ -41,13 +42,9 @@ export default function Datapoint({
         <Text size="sm">{title}</Text>
         {tooltipText && (
           <Tooltip text={tooltipText} idealDirection="up">
-            <IconButton
-              accessibilityLabel=""
-              size="sm"
-              icon="info-circle"
-              iconColor="gray"
-              padding={1}
-            />
+            <TapArea accessibilityLabel="" rounding="circle" tapStyle="none">
```

Comment:
Can you add a comment here that the accessibilityLabels are taken care of by the Tooltip?

Reply:
DONE